src/main/kotlin/org/rust/lang/core/psi/ext/RsItemsOwner.kt

```diff
@@ -139,14 +146,24 @@ private fun lowerImports(imports: List<RsUseItem>): NamedAndStarImports {
  */
 class RsCachedItems(
     private val imports: Lazy<NamedAndStarImports>,
-    /** [RsMacro2] are stored in [named] */
+    /** May contain cfg-disabled items. [RsMacro2] are stored in [named] */
     val legacyMacros: List<RsMacro>,
+    /** May contain cfg-disabled items*/
     val named: Map<String, List<RsItemElement>>,
```

Comment:
I guess we need to add cfg check to:
* https://github.com/intellij-rust/intellij-rust/blob/e27baa8118171b6bc7d4888c5313d2bc5f5b3769/src/main/kotlin/org/rust/lang/core/psi/ext/RsMod.kt#L107-L108
* https://github.com/intellij-rust/intellij-rust/blob/e27baa8118171b6bc7d4888c5313d2bc5f5b3769/src/main/kotlin/org/rust/lang/core/resolve/ItemResolution.kt#L83